test/fuzz/meson.build

```diff
@@ -7,7 +7,7 @@ sanitize_address_undefined = custom_target(
                    project_source_root,
                    '@OUTPUT@',
                    'fuzzers',
-                   '-Db_lundef=false -Db_sanitize=address,undefined',
+                   '-Db_lundef=false -Db_sanitize=address,undefined --optimization=1',
```

Comment:
I wonder if it would be better to replace it with `'... --optimization=@0@'.format(get_option('optimization'))`.

Reply:
That's unquestionably better, thanks. I really need to get into meson properly anytime soon :-)